test/test_optim.py

```diff
@@ -675,6 +647,22 @@ def new_pattern():
 
         self.assertWarnsRegex(new_pattern, r'`optimizer.step\(\)` has been overridden')
 
+    def _test_lr_is_constant_for_constant_epoch(self, scheduler):
+        l = []
+
+        for _ in range(10):
+            scheduler.step(2)
+            l.append(self.opt.param_groups[0]['lr'])
+        self.assertAlmostEqual(min(l), max(l))
```

Comment:
Not just almost equal, exactly equal, right?!

Reply:
The `step` operator will recompute the value at each iteration, and indeed the two real numbers should be equal, yes. I can replace by `assertEqual` if we are not worried about machine precision.

Reply:
The tests pass with `assertEqual`, so I've updated the tests.